src/components/inputCheckbox/AposCheckbox.vue

```diff
@@ -31,13 +31,19 @@ export default {
     id: String,
   },
   methods: {
-    readOnly() {
-      event.preventDefault();
-      event.stopPropagation();
-      this.$emit('toggle', this.choice.value);
+    readOnly(event) {
+      this.$nextTick(() => {
+        event.preventDefault();
+        event.stopPropagation();
+        console.log('readonlying');
+        this.$emit('toggle', this.choice.value);
+      })
     },
     toggle(event) {
-      this.$emit('toggle', this.choice.value);
+      this.$nextTick(() => {
+        console.log('toggling');
```

Comment:
stray log

Reply:
actually meant to back out these `nextTicks`s ..